MeshLib/MeshGenerators/VtkMeshConverter.cpp

```diff
@@ -250,7 +260,14 @@ MeshLib::Mesh* VtkMeshConverter::constructMesh(const double* pixVal,
 	if (elements.empty())
 		return nullptr;
 
-	return new MeshLib::Mesh("RasterDataMesh", nodes, elements); // the name is only a temp-name, the name given in the dialog is set later
+	if (value_vec->empty())
+		properties.removePropertyVector("Colour");
+
+	boost::optional< MeshLib::PropertyVector<unsigned>& > materials =
+		properties.createNewPropertyVector<unsigned>("MaterialIDs", MeshLib::MeshItemType::Cell, 1);
+	materials->insert(materials->begin(), elements.size(), 0);
```

Comment:
it returns an empty vector. i changed to insert location to ```materials->end()``` anyway to keep it consistent with other cases where this is done.